docs/mp/integrations/wls.adoc

```diff
@@ -0,0 +1,472 @@
+your application build.
+
+[source, shell]
+.Example of installing thin T3 client artefact to the local maven repository
+----
+mvn install:install-file \
+-Dfile=<JAR_FILE_PATH>/wlthint3client.jakarta.jar \
+-DgroupId=wlthint3client.jakarta \
+-DartifactId=wlthint3client-jakarta \
+-Dversion=1.0
+----
+
+=== Maven Coordinates [[JMS-Integration-Maven-Coordinates]]
+
+To enable Oracle WebLogic Server JMS  with Helidon, add the following dependencies to your project’s `pom.xml` file:
+
+[source,xml]
+.Dependencies for Reactive Messaging with thin T3 client
+----
+<dependency>
+   <groupId>io.helidon.microprofile.messaging</groupId>
+   <artifactId>helidon-microprofile-messaging</artifactId> <1>
+</dependency>
+<dependency>
+    <groupId>io.helidon.messaging.jms</groupId>
+    <artifactId>helidon-messaging-jms</artifactId> <2>
+</dependency>
+<dependency>
+    <groupId>wlthint3client.jakarta</groupId>
+    <artifactId>wlthint3client-jakarta</artifactId> <3>
+    <version>1.0</version>
+</dependency>
+----
+<1> Dependency for Reactive Messaging
+<2> Dependency for JMS connector
+<3> Dependency for manually installed WLS thin client
+
+
+=== Usage [[JMS-Integration-Usage]]
+
+After adding the Maven dependencies,
+configure the Helidon JMS connector including the JMS environment properties and the JMS resources such as the connection factory,
+destination, and destination type.
+
+The following example shows the `helidon-jms` connector configuration in the `application.yaml` configuration file.
+
+[source, yaml]
+.Example messaging configuration with JMS connector and thin T3 client
+----
+wls-username: weblogic
+wls-password: welcome1
+wls-admin-url: t3://localhost:7001 #<1>
+wls-cluster-url: t3://localhost:7003,localhost:7005,localhost:7007 #<2>
+
+mp:
+  messaging:
+    connector:
+      helidon-jms:
+        jndi:
+           jms-factory: qcf #<3>
+           env-properties: #<4>
+              java.naming.factory.initial: weblogic.jndi.WLInitialContextFactory
+              java.naming.provider.url: ${wls-admin-url}
+              java.naming.security.principal: ${wls-username}
+              java.naming.security.credentials: ${wls-password}
+
+    incoming:
+      from-wls-q:
+        connector: helidon-jms
+        jndi.destination: queuejndi #<5>
+        type: queue
+
+    outgoing:
+      to-wls-q:
+        connector: helidon-jms
+        jndi.destination: queuejndi #<5>
+        type: queue
+----
+<1> Admin server t3 connection URL
+<2> Example of WLS Cluster t3 connection URL
+<3> Connection factory name
+<4> JMS environment properties to lookup resources
+<5> Destination with `jndi.` prefix is evaluated as JNDI name, simple `destination` is evaluated as a CDI syntax
+
+For more detailed reading about Reactive Messaging configuration see xref:../../mp/reactivemessaging/introduction.adoc#_configuration[Reactive Messaging Configuration].
```

Comment:
Please consider adding a x-ref to the WLS documentation that describes how to obtain the WL-JMS specific values.